cluster/manifests/deletions.yaml

```diff
@@ -1,9 +1,6 @@
 # everything defined under here will be deleted before applying the manifests
-pre_apply:
+pre_apply: []
 {{ if eq .LifecycleStatus "ready" }}
```

Comment:
Please drop the whole block, it's not needed anymore.

Reply:
done